test/test-dbPsychologists.js

```diff
@@ -80,31 +81,75 @@ describe('DB Psychologists', () => {
     });
   });
 
-  describe("getPsychologistByEmail", () => {
+  describe("getAcceptedPsychologistByEmail", () => {
     it("should return a psy if we enter a known login email", async () => {
       await dbPsychologists.savePsychologistInPG(psyList);
-      const psy = await dbPsychologists.getPsychologistByEmail(psyList[0].personalEmail);
+      const psy = await dbPsychologists.getAcceptedPsychologistByEmail(psyList[0].personalEmail);
       psy.email.should.be.equal(psyList[0].email);
       psy.personalEmail.should.be.equal(psyList[0].personalEmail);
     });
 
     it("should return undefined if we enter a unknown email", async () => {
-      const unknownPsy = await dbPsychologists.getPsychologistByEmail("unknown@unknown.org")
+      const unknownPsy = await dbPsychologists.getAcceptedPsychologistByEmail("unknown@unknown.org")
 
       assert(undefined === unknownPsy);
     });
 
     it("should return undefined if we dont use 'personalEmail' but 'email' as the login", async () => {
       await dbPsychologists.savePsychologistInPG(psyList);
-      const unknownPsy = await dbPsychologists.getPsychologistByEmail(psyList[0].email);
+      const unknownPsy = await dbPsychologists.getAcceptedPsychologistByEmail(psyList[0].email);
       assert(undefined === unknownPsy);
     });
 
     it("should return undefined if known email but not accepte state", async () => {
-      psyList[0].state = 'en_construction';
+      psyList[0].state = demarchesSimplifiees.DOSSIER_STATE.sans_suite;
       await dbPsychologists.savePsychologistInPG(psyList);
-      const unknownPsy = await dbPsychologists.getPsychologistByEmail(psyList[0].personalEmail);
+      const unknownPsy = await dbPsychologists.getAcceptedPsychologistByEmail(psyList[0].personalEmail);
       assert(undefined === unknownPsy);
     });
+    it("should return undefined if known email but not accepte state", async () => {
```

Comment:
Renommer les tests, pour qu'ils aient pas le meme nom, c'est plus facile à suivre

```suggestion
    it("should return undefined if known email but en_construction state", async () => {
```